README.md

```diff
@@ -77,22 +89,132 @@ In the sub-folders for plots that intersected with the orthomosaic, there will b
+    season: 'S7_20181011'
+    germplasmName: Sorghum bicolor
+    collectingSite: Maricopa
+    observationTimeStamp: '2018-10-11T13:01:02-08:00'
+```
+
+Step 1 requires creating two named Docker volumes to use when processing.
+If you already have one or more empty named volumes you can skip this step.
+```bash
+docker volume create my_input
+docker volume create my_output
+``` 
+
+Step 2 involves copying the drone images into a folder:
+```bash
+mkdir -p /inputs
+cp /IMG/* /inputs/
+```
+
+Step 3 copies the optional shapefile files into the same folder as the drone images:
+```bash
+cp /plot_shapes.* /inputs/
+``` 
+
+In step 4 we copy the experiment.yaml file into the same folder as the drone images:
+```bash
+cp experiment.yaml /inputs/
+``` 
+
+In step 5 we copy the source files onto the input named volume:
+```bash
+docker run --rm -v /inputs:/sources -v my_input:/input --entrypoint bash agdrone/canopycover-workflow:latest -c 'cp /sources/* /input/'
+``` 
+
+In step 6 we create a local folder to hold the output from processing:
+```bash
+mkdir -p /output
+```
+
+In step 7 we run the workflow to generate the orothomosaic image using ODM (OrthoDroneMap) and calculate plot-level canopy cover:
+```bash
+docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v /inputs:/scif/data/odm/images -v scif_output:/output -e INPUT_VOLUME=my_input -e OUTPUT_VOLUME=my_output -e "INPUT_IMAGE_FOLDER=/images" -e "OUTPUT_FOLDER=/output" agdrone/canopycover-workflow:latest run odm_workflow plot_shapes.shp my_input my_output
+```
+and we wait until it's finished.
+
+In step 8 we copy the results off the named output volume to our local folder:
+```bash
+docker run --rm -v /output:/results -v my_output:/output --entrypoint bash agdrone/canopycover-workflow:latest -c 'cp -r /output/* /results/'
+```
+The results of the processing are now in the `/output` folder.
+
+Finally, in step 9 we clean up the named volumes by deleting everything on them:
+```bash
+docker run --rm -v my_input:/input -v my_output:/output --entrypoint bash agdrone/canopycover-workflow:latest -c 'rm -r /input/* && rm -r /output/*'
+```
+
 ### Clean
+
 By executing the [scif](#scif) app named `clean` it's possible to clean up the output folder and other generated files.
 It's recommended, but not necessary, to run the clean app between processing runs by either running this command or through other means.
 
 **Example:**
 
-This docker command line will clean up the output files generated using the [Canopy Cover: Orthomosaic and Shapefile](#can_om_shp) example above.
+This docker command line will clean up the output files generated using the [Canopy Cover: Orthomosaic and Shapefile](#om_can_shp) example above.
 ```bash
-docker run --rm -v ~/inputs:/scif/data/odm/images -v ~/outputs:/scif/data/soilmask agdrone/canopycover-shape-workflow:latest run clean
+docker run --rm -v /inputs:/scif/data/odm/images -v /outputs:/scif/data/soilmask agdrone/canopycover-shape-workflow:latest run clean
```

Comment:
Inconsistent Docker image name. Sometimes it's `agdrone/canopycover-shape-workflow` and sometimes it's `agdrone/canopycover-workflow`. Or am I missing something?

Reply:
Nope. It should be `agdrone/canopycover-workflow`